UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -179,6 +179,42 @@ public static void LogMessage(string message)
             Debug.Log(message);
         }
 
+        public static IEnumerator SendInfo()
+        {
+            string payload = _jsonInstance.SerializeObject(new GSDKInfo());
+            if (string.IsNullOrEmpty(payload) || string.IsNullOrEmpty(_baseUrl))
+            {
+                yield break;
+            }
+
+            byte[] payloadBytes = Encoding.UTF8.GetBytes(payload);
+
+            if (IsDebugging)
+            {
+                Debug.Log($"state: {CurrentState}, info payload: {payload}");
+            }
+
+            using (UnityWebRequest req = new UnityWebRequest(_baseUrl, UnityWebRequest.kHttpVerbPOST))
+            {
+                req.SetRequestHeader("Accept", "application/json");
+                req.SetRequestHeader("Content-Type", "application/json");
+                req.uploadHandler = new UploadHandlerRaw(payloadBytes) { contentType = "application/json" };
+                yield return req.SendWebRequest();
+
+                if (req.result == UnityWebRequest.Result.ConnectionError || req.result == UnityWebRequest.Result.ProtocolError)
+                {
+                    Guid guid = Guid.NewGuid();
+                    Debug.LogFormat("Error sending info: {0} - {1}", req.error, guid.ToString());
+                    IsProcessing = false;
+                }
+                else
+                {
+                    CurrentErrorState = ErrorStates.Ok;
+                    IsProcessing = false;
```

Comment:
Is this being used somewhere?

Reply:
Yes in the ...AgentView. The update loop waits for the agent to be done processing before sending the next heartbeat.